### ISSUE TITLE
Bad copy/paste crashed game, now blueprint is potentially broken.

### ISSUE THREAD

**Issue by Vicomt** [tsunamayo/Starship-EVO#3013]
### Short Description of the issue:
I was trying out a copy/paste operation on a ship, it didn't appear to work, but must have done something, as the game crashed. upon reload, the ship appeared to be fine, but the size as shown in the codex and through the ship computer, is incorrect. whereas before, the ship was 144m long, it was now displayed as 544m long, likewise the width and height are incorrect.
Also, when the ship is flown, the viewpoint (in both internal and external views) is waaaaaay out behind the ship.

The ship in question is called "Fenrir" - and should have been renamed (in the savegame) to "Fenrir Bad"

### Steps to reproduce the issue:
I don't know? it involved copy/paste anyway.
...

### Environment
- Starship EVO version: 20w40a
- Graphics Card: NVIDIA GTX 1070

### Additional information:
exported blueprint
[Fenrir bad.zip](https://github.com/tsunamayo/Starship-EVO/files/5308187/Fenrir.bad.zip)

savegame
[Vic.zip](https://github.com/tsunamayo/Starship-EVO/files/5308192/Vic.zip)

**Comment by tsunamayo** [comment]
@Vicomt you dont have the output.log file? I need it to understand what happened. What were you copying? Thanks

**Comment by Vicomt** [comment]
no, unfortunately, I don't have the log file. I suppose all I really need to know is whether that blueprint is broken now or not. if it's broken I'll need to revert to an earlier copy without some sizeable changes.

**Comment by tsunamayo** [comment]
yeah you just want to fix your ship, I just want to fix my game ;)

**Comment by tsunamayo** [comment]
Did you copied some engine by any chance? thanks

**Comment by Vicomt** [comment]
it's possible I did. I was playing with copy/paste and "something" happened, my selection box was some kind of weird plane shooting off into space, there was a large FPS hitch, then nothing, normal operation. we didn't discover the blueprint size issue until after we'd restarted.
